src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  text-transform: uppercase;
+  font-size: 12px;
+  width: 100%;
+
+  --bluelinkcolor: #00acdc;
 }
+
+.header {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  list-style: none;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  text-decoration: none;
+  display: inline-block;
+  margin-right: 20px;
+  color: black;
+  height: 60px;
+  line-height: 60px;
+}
+
+.nav__link--last {
```

Comment:
better use this:

```
.nav_item:not(:last-child) {
  margin-right: 20px
}
```

**`There should not be margins before the first and after the last list items`**